audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/service/LoggerAuditService.java

```diff
@@ -96,11 +105,12 @@ public Boolean audit(final String token, final AuditMessage message) {
             if (message.getServiceName().equals(ServiceName.FILTERED_RESOURCE_SERVICE.value) || message.getServiceName().equals(ServiceName.DATA_SERVICE.value)) {
                 auditSuccessMessage(auditLogger, successMessage);
                 return true;
-            } else {
-                auditLogger.warn("An AuditSuccessMessage should only be sent by the filtered-resource-service or the data-service. Message received from {}",
-                        message.getServiceName());
-                return false;
             }
+            auditLogger.warn(
+                "An AuditSuccessMessage should only be sent by the filtered-resource-service or the data-service. Message received from {}",
```

Comment:
capitalisation of services

Reply:
yeah, references to our services should be Capitalised, "Filtered Resource Service"

Reply:
Cool. Updated